src/helpers/component-tree.ts

```diff
@@ -4,9 +4,7 @@ import { ReactTestInstance } from 'react-test-renderer';
  * Checks if the given element is a host element.
  * @param element The element to check.
  */
-export function isHostElement(
-  element?: ReactTestInstance | null
-): element is ReactTestInstance {
+export function isHostElement(element?: ReactTestInstance | null) {
```

Comment:
you don't need the the typeguard anymore?

Reply:
I've added the type guard temporary in this PR but it caused more trouble than benefit, as in negative case TS assumed that element is `null`, which is not the case.

Reply:
oh ok i understand